db/schema.rb

```diff
@@ -246,6 +246,7 @@
     t.datetime "welcome_email_date_utc"
     t.datetime "invite_date_utc"
     t.datetime "accept_terms_date_utc"
+    t.string "aasm_state"
```

Comment:
`aasm_state` is the default. Can change to `state` just means adding a line of code into the model to look for this instead of the default; I was going for less code but happy to update

Reply:
out of the box it be then

aasm => adding a state machine